deploy-manage/remote-clusters/_snippets/rcs-elasticsearch-api-snippet.md

```diff
@@ -0,0 +1,43 @@
+<!--
+This snippet is in use in the following locations:
+- ec-remote-cluster-same-ess.md
+- ec-remote-cluster-other-ess.md
+- ec-remote-cluster-ece.md
+- ece-remote-cluster-same-ece.md
+- ece-remote-cluster-other-ece.md
+- ece-remote-cluster-ess.md
+
+It requires remote_type substitution to be defined
+-->
+To configure a deployment as a remote cluster, use the [cluster update settings API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-cluster-put-settings). Configure the following fields:
+
+* `Remote cluster alias`: When using API key authentication, the cluster alias must match the one you configured when adding the API key in the Cloud UI as **Remote cluster name**.
```

Comment:
I still suggest that these become markup on the codeblock. it's difficult to understand that this is the name of the `remote` object you're adding in the call. either do that, or maybe clarify above the code snippet, saying something like

```
This is an example of the API call to `_cluster/settings`, where the remote cluster alias matches the remote cluster name `alias-for-my-remote-cluster`.
```

if you really don't think it's needed you don't have to do it. just my opinion :)

Reply:
hopefully solved :)